utils.groovy

```diff
@@ -959,4 +959,48 @@ def build_and_push_image(params = [:]) {
     return manifest_digest
 }
 
+def get_arch_image_digest(from, arch) {
+    return shwrapCapture("""\
+        skopeo inspect --raw docker://${from} | \
+        jq -r '.manifests[] | select(.platform.architecture == "${arch}" and .platform.os == "linux") | .digest'
+    """.stripIndent())
+}
+
+def brew_upload(arches, release, repo, manifest_digest, extensions_manifest_digest, version, pipecfg) {
+    def brew_principal = pipecfg.brew.principal
+    def brew_profile = pipecfg.brew.profile
+    def tag = pipecfg.streams[release].brew_tag
+    def archAliasMap = [
+        "x86_64"  : "amd64",
+        "aarch64" : "arm64",
+        "s390x"   : "s390x",
+        "ppc64le" : "ppc64le"
+    ]
+    for (arch in arches) {
+        inspect_arch = archAliasMap[arch]
+        image_digest = get_arch_image_digest("${repo}@${manifest_digest}", inspect_arch)
+        extensions_image_digest = get_arch_image_digest("${repo}@${extensions_manifest_digest}", inspect_arch)
```

Comment:
Since we're doing this in a loop, it'd be more efficient to query the manifests once and build a table of `arch -> digest`.